src/shared/api/validation.ts

```diff
@@ -324,13 +361,21 @@ export function reportValidationResult(result: any, prefix = '') {
         });
 
     result.status &&
-        console.log(
-            `${prefix} ${result.label} (${
-                result.hash
-            }) passed :) ch: ${result.chDuration.toFixed(
-                0
-            )} legacy: ${result.legacyDuration.toFixed(0)}`
-        );
+        (!result.legacyDuration
```

Comment:
try to avoid these duplicating logic (console.log(...)) like this for only a "little" thing like whether it has legacyDuration.  better to introduce a variable that gets always gets printed and can sometimes be blank (e.g. when there's no legacyDuration).